examples/manifest_script_template.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env -S -i CM_PROD_DIR="${CM_PROD_DIR}" HOME="${HOME}" bash
+
+# The shebang lines above are needed b/c setup lsst_distrib in putting
+# the lsst python _after_ the virtual env python in the PATH, which
+# is causing errors
+
+# setup LSST env.
+export WEEKLY='{lsst_version}'
+source /cvmfs/sw.lsst.eu/linux-x86_64/lsst_distrib/${WEEKLY}/loadLSST.bash
+setup lsst_distrib
```

Comment:
Does manifest checking happen in this script or somewhere else?

Reply:
This is the wrapper that sets up the env for that script.  That script gets defined by the ManfestReportScriptHandler, here: https://github.com/lsst-dm/cm-service/blob/05a9a04824801e72b1f20607406d3a49d0db9d4a/src/lsst/cmservice/handlers/jobs.py#L292